### PR TITLE
Typo "Microsoft Teams Room"→"Microsoft Teams Rooms"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://docs.microsoft.com/en-us/microsoftteams/rooms/rooms-deploy
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/rooms/rooms-deploy.md
#PingMSFTDocs

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @hyoshioka0128, kindly commit the minor suggested change. Many thanks.

File: Teams/rooms/rooms-deploy.md [hunk 1]
```diff
@@ -48,12 +48,12 @@ You can find out more about these dependencies in the planning guidance links be
...
+-   If joining Teams Rooms to Azure Active Directory, prepare an Azure AD group with dynamic membership to hold all of the Teams Rooms resource accounts. This will simplify future management, such as applying Conditional Access policies. In order to most easily leverage Azure AD dynamic groups, determine a naming convention that will uniquely identify your Teams Rooms resource accounts.
+-   If joining Teams Rooms to Active Directory, prepare an organizational unit and Active Directory group to hold your Microsoft Teams Rooms machine and resource accounts, and—optionally—prepare Group Policy objects (GPOs) to enable PowerShell remoting.
 
 ### Define Microsoft Teams Rooms resource account features 
 
-Depending on the collaboration scenarios that you've decided to enable with your Microsoft Teams Rooms deployment, you'll need to determine the features and capabilities that you assign to each Microsoft Teams Room that you enable.
+Depending on the collaboration scenarios that you've decided to enable with your Microsoft Teams Rooms deployment, you'll need to determine the features and capabilities that you assign to each Microsoft Teams Rooms that you enable.
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-deploy.md:56]:
```suggestion
Depending on the collaboration scenarios that you've decided to enable with your Microsoft Teams Rooms deployment, you'll need to determine the features and capabilities that you assign to each Microsoft Teams room that you enable.
```

**Comment by hyoshioka0128** [comment]
@JohanFreelancer9 - Thanks!

**Comment by scanum** [comment]
@cazawideh PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**